### PR TITLE
Backend configuration with support tools

### PR DESCRIPTION
**Pull request by mateuszzalewski011**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/functiona_feature_request.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+name: Syntax feature request
```
Comment (Argmaster) [inline .github/ISSUE_TEMPLATE/functiona_feature_request.md:2]:
```suggestion
name: Feature request
```

File: .github/ISSUE_TEMPLATE/functiona_feature_request.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+title: ""
+labels: "enhancement"
+assignees: ""
+---
+
+**_I have verified that this feature is not listed as implemented in README.md on `main`
+branch_** (yes/no)
+
+---
+
```
Comment (Argmaster) [inline .github/ISSUE_TEMPLATE/functiona_feature_request.md:13]:
```suggestion

```

Comment (Argmaster) [inline .github/ISSUE_TEMPLATE/functiona_feature_request.md:13]:
```suggestion
```

File: test/__init__.py [hunk 3]
```diff
@@ -0,0 +1 @@
+"""PEC Framework test suite root."""
```
Comment (Argmaster) [inline test/__init__.py:1]:
```suggestion
"""Epseon GUI test suite root."""
```

**Review by Argmaster — CHANGES_REQUESTED** [review]

**Comment by Argmaster** [comment]
I'd be great to have those fixed before next meeting, maybe even merged before.

File: .dockerignore [hunk 4]
```diff
@@ -0,0 +1,2 @@
+Dockerfile*
```
Comment (Ave44) [inline .dockerignore:1]: If we'll change name of our dockerfiles then we should add rule `*.dockerfile`

**Review by Ave44 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/functiona_feature_request.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+name: Feature request about: Request new functional feature. title: "" labels:
```
Comment (Argmaster) [inline .github/ISSUE_TEMPLATE/functiona_feature_request.md:3]: You have messed up header structure.

File: .pre-commit-config.yaml [hunk 6]
```diff
@@ -0,0 +1,37 @@
+exclude: '^(\.tox|\.env|dist|\.vscode)(/|$)'
+repos:
+    - repo: https://github.com/pre-commit/mirrors-prettier
+      rev: "v2.7.1"
```
[the remark this answers is not in the document]
Comment (Argmaster) [reply .pre-commit-config.yaml:4]: @mateuszzalewski011 please change this to `v3.0.3`

[the remark this answers is not in the document]
Comment (Argmaster) [reply .pre-commit-config.yaml:4]: Okay, but it still shows as `v2.7.1` in review, am I missing something?

**Comment by Argmaster** [comment]
Any chance to get this merged? I am still waiting for response on Prettier version.

File: .pre-commit-config.yaml (hunk 6 above)
[the remark this answers is not in the document]
Comment (Ave44) [reply .pre-commit-config.yaml]: version updated

**Review by Ave44 — DISMISSED** [review]

**Review by Argmaster — DISMISSED** [review]

**Review by Argmaster — APPROVED** [review]

**Review by Ave44 — APPROVED** [review]

**Pull request merged**